diamond-api/src/render.rs

```diff
@@ -144,18 +151,33 @@ fn walk(dir: &Path, metric: &str, interval: Interval) -> Result<Vec<RenderPoint>
     Ok(points)
 }
 
+#[derive(Clone)]
```

Comment:
Looks like Clone is not needed here. `walker` function consumes value and makes no clones.

Reply:
`fn walker(w: Walker) -> ...` has to consume Walker (thus I pass cloned value), otherwise:
`cannot infer an appropriate lifetime`, as borrowed reference is moved into the closure.

Reply:
Now I see.